app/views/sufia/admin/admin_sets/index.html.erb

```diff
@@ -27,6 +28,7 @@
                     <td><%= link_to admin_set.title.first, [sufia, :admin, admin_set] %></td>
                     <td><%= admin_set.create_date %></td>
                     <td><%= safe_join(admin_set.creator) %></td>
+                    <td><%= @presenter.total_items(admin_set) %></td>
```

Comment:
The presenter is an object that presents a record. In this case that record should be `admin_set`, so you can either declare it here or in the controller, but there should be no need to pass arguments to it's methods.